LBHTenancyAPITest/Test/Gateways/V2/ArrearsActions/UHArrearsDiaryCreationGatewayTest.cs

```diff
@@ -139,7 +163,7 @@ public async Task UpdateRecordingUserNameWhenNoUsernameSupplyed_ShouldNoOp(strin
             IArrearsActionDiaryGateway classUnderTest = new ArrearsActionDiaryGateway(fakeArrearsAgreementService.Object, _databaseFixture.ConnectionString);
 
             //act
-            await classUnderTest.UpdateRecordingUserName(username, diaryEntry.Id);
+            await classUnderTest.UpdateRecordingDetails(username, diaryEntry.Id, DateTime.Now);
             //assert
             ArrearsActionDiaryEntry action = TestDataHelper.GetArrearsActionsByRef(diaryEntry.TenancyRef).First();
 
```

Comment:
An assert should be made here to test that date has been set when UniversalHousingUsername was left null
```suggestion
action.Date.Should().Be(date);
```

Reply:
Added the test as suggested.